ooniprobe/View/RunTestViewController.mm

```diff
@@ -38,7 +43,9 @@ - (void)viewDidLoad {
     else {
         [self.tableView setSeparatorStyle:UITableViewCellSeparatorStyleNone];
     }
-    
+    dispatch_async(dispatch_get_main_queue(), ^{
+        [self.tableView reloadData];
+    });
```

Comment:
I don't get exactly why this is necessary now and was not necessary before.

Reply:
Now XCode throws a warning, before didn't :D

Reply:
Yeah, but before you were not calling `reloadData` at all, no?

Reply:
Oh I thought the comment was about the dispatch_async.
No before I wasn't because I wasn't handling the case of reloading the view with new parameters.

Reply:
oh, right, thanks for explaining!

Reply:
Again, maybe a comment in the code may help.